dev-requirements.txt

```diff
@@ -3,9 +3,6 @@
 # This list will have to be migrated if you wish to use another dependency manager.
 
 # Base tools
-pillow~=8.3.1
-numpy~=1.21.0
-blessed~=1.18.1
```

Comment:
All our requirements have to be in `dev-requirements.txt` I think

Reply:
Yes, but we also have `requirements.txt` that the user will install when using our app. No need for them to install our development requirements.

Reply:
When our project gets reviewed, they will use `dev-requirements.txt` I think. Let's just keep them both

Reply:
leave them in both then, we also need those packages in dev anyway

Reply:
Yeah I fixed that now